### DISCUSSION TITLE
[Question] Can the icon of closeEditor in AppBar be hidden?

### DISCUSSION THREAD

**Discussion by likaia** [hm21/pro_image_editor#248] [Q&A]
I need to hide the closeEditor icon, as shown below:

<img width="1087" alt="image" src="https://github.com/user-attachments/assets/927faf7d-de74-45fc-bbcb-4db6c4ad7538">

**Comment by hm21** [comment]
It seems like you're using the prebuilt frosted glass design, correct? If so, you'll need to replace the prebuilt [ActionBar](https://github.com/hm21/pro_image_editor/blob/54b496080011a07d1fd139ac42b868aea957c015/example/lib/pages/design_examples/frosted_glass_example.dart#L248-L251) with your own implementation. You can copy the default frosted glass action bar from [here](https://github.com/hm21/pro_image_editor/blob/54b496080011a07d1fd139ac42b868aea957c015/lib/designs/frosted_glass/frosted_glass_appbar.dart#L65-L75), and simply replace the close button with a SizedBox.

FYI I've moved this question to the “Discuss” tab, as I prefer to use that tab for Q&A.

**Reply by likaia** [reply]
Thanks for your reply. Let me have a try. However, I don't think there is an option to hide this button in the default configuration. It would be better if it could be added to the configuration item.
